e2e-tests/playwright/e2e/plugins/tekton/tekton.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { test } from '@playwright/test';
+import { Common } from '../../../../playwright/utils/Common';
+import { UIhelper } from '../../../../playwright/utils/UIhelper';
+import { Tekton } from '../../../utils/tekton/tekton';
+
+// Pre-req: Enable tekton, kubernetes, kubernetes-backend plugins
+// Pre-req: install Red Hat OpenShift Pipelines Operator
+// Pre-req: Create a pipeline run
+// Pre-req: A kubernetes cluster containing pipeline and pipelinerun resources labeled with backstage.io/kubernetes-id: developer-hub
+// Pre-req: A catalog entity with the matching backstage.io/kubernetes-id: developer-hub annotation as well as the janus-idp.io/tekton : <BACKSTAGE_ENTITY_NAME> annotation
+
+test.describe('Test Tekton plugin', () => {
+  let common: Common;
+  let uiHelper: UIhelper;
+  let tekton: Tekton;
+
+  test.beforeAll(async ({ browser }) => {
+    const page = await browser.newPage();
+    common = new Common(page);
+    await common.loginAsGuest();
+    uiHelper = new UIhelper(page);
+    tekton = new Tekton(page);
+  });
+
+  test('Check Pipeline Run', async () => {
+    await tekton.goToBackstageJanusProject();
+    await tekton.verifyTableHasRows();
```

Comment:
verifyTableHasRows seems a bit confusing as to which table it is verifying. Can we rename it or pass pipeline runs as a parameter?